src/test/java/uk/gov/companieshouse/web/companydocuments/controller/CompanyDetailsControllerTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.companieshouse.web.companydocuments.controller;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.model;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.view;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import uk.gov.companieshouse.web.companydocuments.model.CompanyDetail;
+import uk.gov.companieshouse.web.companydocuments.service.CompanyService;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CompanyDetailsControllerTest {
+
+    private MockMvc mockMvc;
+
+    @Mock
+    private CompanyService companyService;
+
+    @Mock
+    private CompanyDetail companyDetail;
+
+    @InjectMocks
+    private CompanyDetailsController controller;
+
+    private static final String COMPANY_NUMBER = "companyNumber";
```

Comment:
Constants should come first.

Reply:
Moved in [b916061](https://github.com/companieshouse/company-documents.web.ch.gov.uk/pull/10/commits/b9160619da9c40be5d6b3314aec0733f77b248a6) 👍